container-chains/templates/frontier/runtime/src/xcm_config.rs

```diff
@@ -439,3 +469,17 @@ pub type ForeignFungiblesTransactor = FungiblesAdapter<
 /// Multiplier used for dedicated `TakeFirstAssetTrader` with `ForeignAssets` instance.
 pub type AssetRateAsMultiplier =
     AssetFeeAsExistentialDepositMultiplier<Runtime, WeightToFee, AssetRate, ForeignAssetsInstance>;
+
+#[test]
+fn test_asset_id_to_account_conversion() {
```

Comment:
I think we don't need this test

Reply:
Currently `asset_id_to_account` takes `prefix` as slice, instead of hard bound array. Then it does invoke `copy_from_slice`, which can panic, if the slice's size is not exact. 

If the implementation changes tomorrow (i.e the expected size of the slice in the implementation), it need to be captured by the unit test (otherwise you would see panic during integration test or worse in dev environment). Which this test prevents.

Reply:
Let's discuss this offline. I am merging this as it is not blocker as of now.